### PR TITLE
check aefId twice, fix CamelCase, 404 when no logs, return one InvocationLog

### PR DESCRIPTION
**Pull request by sxarism**

### CODE REVIEW

File: services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py [hunk 1]
```diff
@@ -69,25 +71,44 @@ def add_invocationlog(self, aef_id, invocationlog):
             if result is not None:
                 return result
 
+            current_app.logger.debug("URL aef_id: {}".format(aef_id))
+            current_app.logger.debug("URL invocationlog.aef_id: {}".format(invocationlog.aef_id))
+            if aef_id != invocationlog.aef_id:
```
Comment (raxonros) [inline services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py]: I think its better to move this check in __check_aefid function

File: services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py [hunk 2]
```diff
@@ -69,25 +71,44 @@ def add_invocationlog(self, aef_id, invocationlog):
...
 
-            log_id = secrets.token_hex(15)
-            rec = dict()
-            rec['log_id'] = log_id
-            rec.update(invocationlog.to_dict())
-            mycol.insert_one(rec)
+            my_query = {'aef_id': aef_id, 'api_invoker_id': invocationlog.api_invoker_id}
+            existing_invocationlog = mycol.find_one(my_query)
+
+            if existing_invocationlog is None:
```
Comment (raxonros) [inline services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py:91]: What is the meaning of Test 0 ?

File: services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py [hunk 3]
```diff
@@ -69,25 +71,44 @@ def add_invocationlog(self, aef_id, invocationlog):
...
+                current_app.logger.debug("Test 0")
```
Comment (raxonros) [inline services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py]: Why are you updating log_id?

File: services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py (hunk 1 above)
Comment (sxarism) [reply services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py]: Ok. I will change it.

File: services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py (hunk 2 above)
Comment (sxarism) [reply services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py:91]: No. It was just for debugging

File: services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py (hunk 3 above)
Comment (sxarism) [reply services/TS29222_CAPIF_Logging_API_Invocation_API/api_invocation_logs/core/invocationlogs.py]: I don't update log_id here. I create it. This is the case where no similar InvocationLog is found. So i create the log_id

**Review by raxonros — APPROVED** [review]
LGTM¡

**Pull request merged**